router/protocol.go

```diff
@@ -329,10 +329,13 @@ type ProtocolTag byte
 
 const (
 	ProtocolHeartbeat ProtocolTag = iota
-	ProtocolOverlayControlMsg
+	ProtocolConnectionEstablished
+	ProtocolFragmentationReceived
+	ProtocolPMTUVerified
```

Comment:
That's right, the sleeve message type numbering is likely to retain these values forever .  Though what I had in mind for 1.3 was to move the definitions back to sleeve.go, with their old naming but the values set here.

As for structuring things for a 'git revert': One of the benefits of keeping these compatibility PRs distinct is that it will facilitate the task of dropping 1.1 compatibility in 1.3.  But I don't expect it to be quite as a simple as a 'git revert'.  Even if we did the work now to allow that, other changes might be made between now and 1.3 that would present obstacles. So I think we can cross that bridge when we come to it.